src/lib/pubkey/sphincsplus/sphincsplus_common/sp_parameters.h

```diff
@@ -0,0 +1,235 @@
+/*
+ * SPHINCS+ Parameters
+ * (C) 2023 Jack Lloyd
+ *     2023 Fabian Albert, René Meusel, Amos Treiber - Rohde & Schwarz Cybersecurity
+ *
+ * Botan is released under the Simplified BSD License (see license.txt)
+ **/
+
+#ifndef BOTAN_SP_PARAMS_H_
+#define BOTAN_SP_PARAMS_H_
+
+#include <botan/asn1_obj.h>
+
+#include <string_view>
+
+namespace Botan {
+
+enum class Sphincs_Hash_Type {
+   Shake256,
+   Sha256,
+   Haraka,
```

Comment:
Add a comment here noting that Haraka is unsupported currently as a hint for someone trying to use it and confused why it always fails.